new-style.css

```diff
@@ -908,6 +953,55 @@ html[dir="rtl"] .footer .navs {
     .new-steps .step .description {
         font-size: 0.875em;
     }
+
+    /*Covid*/
+
+    .covid .covid-content .covid-container {
+        display: flex;
+      }
+    
+      .covid .covid-content .covid-container p {
+        color:#fff;
+        text-align: start;
+        padding-left: 0.75em;
+        padding-right: 0.75em;
+      }
+    
+      .covid .covid-content h1 {
+        color:#fff;
+        text-align: center ;
+        padding-inline-start: 0.25em;
+        padding-inline-end: 0.75em;
+      }
+    
+      html[dir="rtl"] .covid .covid-content h1 {
+        color:#fff;
+        right:initial;
+        padding-inline-start: 0.75em;
```

Comment:
Why?

Reply:
what is problem ?

Reply:
Why Hebrew requires 0.75em padding from the start (right) but English doesn't? (has 0.25em from start (left))